mempool README.txt

```diff
@@ -0,0 +1,163 @@
+Raylib Memory Pool
+By Kevin 'Assyrianic' Yonan @ https://github.com/assyrianic
+
+About:
+	The Raylib Memory Pool is a quick, efficient, and minimal free list & stack-based allocator.
+
+Purpose:
+	Raylib Memory Pool's purpose is the following list:
+		* A quicker, efficient memory allocator alternative to `malloc` and friends.
+		* Reduce the possibilities of memory leaks for beginner developers using Raylib.
+		* Being able to flexibly range check memory if necessary.
+
+Data Implementation:
+	the memory pool encapsulates two public structs:
+	`freeList` which is an abstracted doubly linked list consisting of a `head` and `tail` pointer to `struct MemNode`, a `len` that tracks the amount of nodes the linked list holds, `maxNodes` which is used for auto-defragging (explained below), and `autoDefrag` which controls whether auto-defragging will execute or not.
+	```c
```

Comment:
Use a `.md` file extension for markdown files. While at it, drop the space from the filename